### PR TITLE
adding signup button to bottom of commcare homepage

### PR DESCRIPTION
**Pull request by calellowitz**
FogBugz ticket here
http://manage.dimagi.com/default.asp?176938#1010226

signupbuttonscreenshot

@nickpell @biyeun

### CODE REVIEW

**Comment by czue** [comment]
first PR!

**Comment by czue** [comment]
looks pretty clean to me! I'll let @biyeun or @orangejenny give a closer look and merge. Also, if you want any feedback on the UI you can cc @dimagi/product

**Comment by orangejenny** [comment]
Looks reasonable to me, though I'd be inclined to make "Get in Touch with Us" no longer green so we don't have two primary actions.

**Comment by amsagoff** [comment]
Do we need more padding above the singup box? Looks a little sandwiched
with the Help Site text (but may be just me)

**Comment by orangejenny** [comment]
> Do we need more padding above the singup box? Looks a little sandwiched
with the Help Site text (but may be just me)

agree

**Comment by calellowitz** [comment]
i changed the padding to match the row above. is there a correct color for the button itself?

**Comment by calellowitz** [comment]
@orangejenny @amsagoff 
Fixed the relevant UI parts should be good to merge. Here is the new screenshot
signupbuttontake2

**Comment by amsagoff** [comment]
:+1:

**Comment by biyeun** [comment]
Cal, this doesn't follow the grid styling, so it looks a bit out of place will comment in detail on the code

File: templates/prelogin/_sections/home/about_dimagi.html [hunk 1]
```diff
@@ -75,5 +75,11 @@ <h2 class="text-center" style="margin-bottom: 1em;">
                 </p>
             </div>
         </div>
+	<div class="col-sm-8 col-md-6 col-sm-offset-2 col-md-offset-3">
+	  <p>
+	    {% trans "Sign up for CommCare now" %}
+	  </p>
+	  {% include 'prelogin/partials/signup.html' %}
```
Comment (biyeun) [inline templates/prelogin/_sections/home/about_dimagi.html]: this shouldn't be a partial. I don't think you should reuse the same element at the top and the bottom, as the top is formatted specifically for that region

**Comment by biyeun** [comment]
More detailed feedback:
1) The top form should not be copied-and-pasted in the bottom, because the design does not fit. Centering the content when the rest of the content is centered and none of the verticals align makes for a design that doesn't look quite thought out.
2) The call-to-action text is also tiny when it should probably be both syntactically and visually an <h3>. The email input and button itself are also too big considering how busy the rest of that portion of the screen is. The colors, green button and light blue background, will do enough to get the form noticed.

My suggestion is that you make a new form at the bottom. 
- Sign Up for CommCare Now is in an h3 tag.
- the input is input-lg, spanning col-sm-5 (offset all the way from the right by col-sm-2) and the button is btn-lg spanning col-sm-3

**Comment by biyeun** [comment]
@orangejenny also made a good point on having two primary actions going on in this screen. There IS a lot going on. What is the primary action here?

I'm personally of the opinion that the sign up should not be in the bottom as well as the top as it really takes away from the bottom information and reduces the impact of the contact us link, but if it has to be that way, i would style the contact us link with the light blue background (cc-brand-hi I believe), and the text as the blue (cc-brand-mid I think).

**Comment by krsdimagi** [comment]
@biyeun the idea of having another sign up at the bottom is because it takes a lot of scrolling to get through all of the informational content, and we didn't want to have a situation where someone doesn't sign up because the initial button is way at the top after they have gotten to the bottom.

**Comment by biyeun** [comment]
@krsdimagi do you have any analytics supporting this? I think adding a section mirroring the top part a little more at the bottom would be more ideal as it is, this is cramming a LOT of information into one section now with several call to actions, and no clear direction for the user as to whether they should contact us or sign up.

I'm just saying I think this should be a little more thought out (I can give it a pass at the summit) and designed rather than just slapped together because this is the FIRST impression a user is getting and the decisions made should be very intentional (like the rest of the page).

**Comment by biyeun** [comment]
@calellowitz sorry for being so harsh on your first PR. This is just the page that people first see when they come to our site, and I want to make sure that impression is on point! Let me know if you want to pair on the styling here, since I think you might have to do some custom stuff to do this properly. :)

**Comment by calellowitz** [comment]
@biyeun understandable. i would rather it be done right than get a PR for the sake of having a PR. pairing sounds good my styling knowledge is not great

**Comment by krsdimagi** [comment]
@biyeun very good points - it's definitely worth another pass.  This just came out of an easy win from a while ago, and could probably benefit from a bit more planning.

**Comment by biyeun** [comment]
Here's my 2 min thought: cchq-static-v3newfoot

**Comment by krsdimagi** [comment]
@biyeun that looks fantastic!  Looped in @amsagoff and we are in agreement.  I checked out the Hubspot analytics for the call to action, and there is an incredibly low (less than a percent) submission rate for the "get in touch with us" button, so it may be good to have more emphasis on signing up at the end of the page.

**Comment by biyeun** [comment]
@calellowitz what's the timeline for this? can we pair as soon as I get off interrupt (next tuesday)? if it's more urgent, we can find some time this week

**Comment by calellowitz** [comment]
@biyeun my understanding is that next week should be fine so lets plan on that

**Comment by biyeun** [comment]
:+1:  done

**Comment by calellowitz** [comment]
@biyeun committed the changes we made
@dimagi/product 
new screenshot:
finalsignupbutton

**Comment by biyeun** [comment]
pro :+1:

**Comment by czue** [comment]
looking good

**Pull request merged**